skt/executable.py

```diff
@@ -385,7 +385,8 @@ def setup_parser():
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("-d", "--workdir", type=str, help="Path to work dir")
+    parser.add_argument("-d", "--workdir", type=str, help="Path to work dir",
+                        default=os.environ.get("WORKDIR"))
```

Comment:
Could you use name "SKT_WORKDIR" instead, so it plays better with other environment variables?

Reply:
Done in #31